conf/runner.yaml

```diff
@@ -224,12 +241,28 @@ runs:
         - addon.mozilla.org
         - addons.mozilla.org
         - blocklist.addons.mozilla.org
-        - builder.addons.mozilla.org
         - services.addons.mozilla.org
         - static.addons.mozilla.net
         - versioncheck-bg.addons.mozilla.org
         - versioncheck.addons.mozilla.org
         - compatibility-lookup.services.mozilla.com
+        - addons.cdn.mozilla.net
+        - addons-discovery.cdn.mozilla.net
+        - addons-amo.cdn.mozilla.net
+        - addons-dev.allizom.org
+        - addons.allizom.org
+        - discovery.addons-dev.allizom.org
+        - discovery.addons.allizom.org
+        - services.addons-dev.allizom.org
+        - services.addons.allizom.org
+        - versioncheck-dev.allizom.org
+        - versioncheck.allizom.org
+        - addons-dev-cdn.allizom.org
+        - addons-stage-cdn.allizom.org
+        - addons-amo-dev-cdn.allizom.org
+        - addons-amo-cdn.allizom.org
+        - addons-discovery-dev-cdn.allizom.org
+        - addons-discovery-cdn.allizom.org
```

Comment:
Maybe no need to page for the -dev and -stage endpoints? I think the ">35d" email reminder is probably good enough for them.